src/ameba/cli/cmd.cr

```diff
@@ -184,13 +202,79 @@ module Ameba::Cli
   end
 
   private def print_rules(config)
```

Comment:
what about moving this logic to a separate formatter? That way it will be de-coupled from the cli commands and will give an ability to cover it by unit tests

Reply:
Thing is, this functionality covers only the rules and not the sources, so it's not rly suited as a formatter. I could do it, although IMO it would be even more clumsy than what we have here.

Reply:
I would try to decouple the "presentation part" from cmd and keep the latter one as simple as possible. 

If it is not suited for the formatter, maybe we could introduce another abstraction or something?